server/manageuser.php

```diff
@@ -102,6 +102,27 @@ function updateUserPermissions($data){
 	$response = array();
 	$response['success'] = true;
 
+	if($data['userId'] == $USER->getUserId()){
+		$stmt = $DB_CONN->prepare("SELECT lookupName 
+			FROM vita.permission
+				INNER JOIN vita.userpermission ON permission.permissionId = userpermission.permissionId
+			WHERE userPermissionId = ?");
+
+		foreach ($data['removePermissionArr'] as $userPermissionId) {
+			$stmt->execute(array($userPermissionId));
+			$lookupName = $stmt->fetch(PDO::FETCH_ASSOC)['lookupName'];
+
+			if($lookupName == 'edit_user_permission'){
```

Comment:
You shouldn't use the `==` operator for string equality, you should use the `===` operator. https://stackoverflow.com/a/8494892

Reply:
I suppose it's technically fine in this case since we know they both will be strings so there will be no type coercion, but just as a general good practice, I think `===` is better

Reply:
Also take a look at the answer below that one I posted, and then one below that. That second one down is especially interesting.

Reply:
It is better practice. Updated.